taskfiles/deps/utils.yaml

```
@@ -0,0 +1,78 @@
+version: "3"
+
+includes:
+  yscope-dev-utils: "../../tools/yscope-dev-utils/exports/taskfiles/utils/utils.yaml"
+
+tasks:
+  # Cleans up the core checksum files any time the taskfiles change. This is to ensure that if any
+  # of core's dependencies are removed from the taskfiles, then the checksum files for the removed
+  # dependencies will also be removed. Since we can't easily remove just the checksum files for the
+  # removed dependencies, we remove all of them.
+  clean-old-core-checksum-files:
+    internal: true
+    run: "once"
+    sources:
+      - "{{.ROOT_DIR}}/Taskfile.yaml"
+      - "{{.ROOT_DIR}}/taskfiles/**/*.yaml"
+      - "{{.ROOT_DIR}}/tools/yscope-dev-utils/exports/taskfiles/**/*.yaml"
+    cmd: "rm -f '{{.G_DEPS_CORE_CHECKSUMS_DIR}}'/*"
+
+  combine-core-checksum-files:
```

```suggestion
  # Generates the combined checksum file `G_DEPS_CORE_CHECKSUM_FILE` by concatenating all the core
  # checksum files.
  combine-core-checksum-files:
```
Nit: Might be worth to add a docstring to point out the output file?